File: src/main/java/constants/header/HeaderEntrypoint.java
package constants.header;

public enum HeaderEntrypoint {
    LANDING_PAGE("Landing page"),
    FLIGHTS("Flights"),
    HOTELS("Hotels"),
    TOURS("Tours"),
    CARS("Cars"),
    VISA("Visa"),
    BLOGS("Blogs");

    private final String name;

    HeaderEntrypoint(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
}

File: ts-playwright-test-automation-framework/src/main/code/constants/header/HeaderEntrypoint.ts
export class HeaderEntrypoint {
    static readonly LANDING_PAGE = new HeaderEntrypoint("Landing page");
    static readonly FLIGHTS = new HeaderEntrypoint("Flights");
    static readonly HOTELS = new HeaderEntrypoint("Hotels");
    static readonly TOURS = new HeaderEntrypoint("Tours");
    static readonly CARS = new HeaderEntrypoint("Cars");
    static readonly VISA = new HeaderEntrypoint("Visa");
    static readonly BLOGS = new HeaderEntrypoint("Blogs");

    private constructor(
        private readonly name: string
    ) { }

    getName(): String {
        return this.name;
    }
}